tests/openebs-control-plane-chaos_test.go

```diff
@@ -241,7 +241,7 @@ var _ = Describe("BDD of openebs control plane chaos experiment", func() {
 			chaosEngine, err := clientSet.ChaosEngines("openebs").Get(engineName, metav1.GetOptions{})
 			Expect(err).To(BeNil(), "Fail to get the chaosengine while updating the result in a table")
 			testVerdict := string(chaosEngine.Status.Experiments[0].Verdict)
-			err = utils.UpdateResultTable(experimentName, testVerdict, engineName, clientSet)
+			err = utils.UpdateResultTable("OpenEBS control plane chaos test the control plane component", testVerdict, engineName, clientSet)
```

Comment:
Any specific reason for replacing experimentName var with the hardcoded names (should be ok as we are placing this with exp specific go file and not a common lib file), but curious.

Reply:
Ok, got it. Its a description, not so much a name - instead it is a line so that the readme is more understandable.